packages/cli/src/commands/generate/store.ts

```diff
@@ -0,0 +1,61 @@
+import { Command } from '@oclif/core';
+import { t } from 'i18next';
+import inquirer from 'inquirer';
+import * as path from 'path';
+import { getIntegration } from '../../domains/integration';
+import { getProjectName } from '../../domains/project-name';
+import { cloneGitRepository, terminateGitRepository } from '../../domains/git-repository';
+import { existsDirectory } from '../../domains/directory';
+
+export default class GenerateStore extends Command {
+  static override description = t('command.generate_store.description');
+
+  static override examples = ['<%= config.bin %> <%= command.id %>'];
+
+  static override flags = {};
+
+  static override args = [];
+
+  async run(): Promise<void> {
+    const projectName = await getProjectName(t('command.generate_store.input.project_name'));
+
+    const integration = await getIntegration({
+      message: t('command.generate_store.input.integration'),
+      customIntegrationRepositoryMessage: t('command.generate_store.input.custom_integration_repository')
+    });
+
+    const projectDir = path.join(process.cwd(), projectName);
+
+    if (await existsDirectory(projectDir)) {
+      const { overwrite } = await inquirer.prompt<{ overwrite: boolean }>({
+        type: 'confirm',
+        name: 'overwrite',
+        message: () => t('command.generate_store.input.overwrite', { projectName })
+      });
+
+      if (!overwrite) {
+        this.log(t('command.generate_store.message.skipping'));
+        this.exit(0);
+      }
+    }
+
+    await cloneGitRepository({
+      projectDir,
+      gitRepositoryURL: integration.gitRepositoryURL
+    });
+
+    await terminateGitRepository(projectDir);
+
+    this.log(t('command.generate_store.message.success', { projectName }));
+
+    if (integration.documentationURL) {
+      this.log(
+        t('command.generate_store.message.check_docs', {
+          documentationURL: integration.documentationURL
+        })
+      );
```

Comment:
Can we add a little bit more information like below?

```
To finish the installation, run the following commands:

cd {{ projectName }}
npm install

Then, go to the {{ installationURL }} to learn how to configure the project.

Once finished, you can start the project using this command:

npm run dev
```

`{{ projectName }}` should be the relative path to the project, and `{{ installationURL }}` should be a direct URL to the installation guide for selected integration, not to the whole integration docs.

Ideally, the commands should be displayed in different colors to make them stand out, like in Vue CLI:

![Przechwytywanie](https://user-images.githubusercontent.com/4145208/155957575-3b78beb9-11a8-4545-9fb3-a9c548219ab8.JPG)